src/main/java/pro/quicksense/service/PaymentService.java

```diff
@@ -0,0 +1,237 @@
+
+        JSONObject bankDetails = new JSONObject();
+        bankDetails.put("account_currency", "HKD");
+        bankDetails.put("account_name", "John Walker");
+        bankDetails.put("account_number", "786005728434");
+        bankDetails.put("account_routing_type1", "bank_code");
+        bankDetails.put("account_routing_value1", "024");
+        bankDetails.put("bank_country_code", "HK");
+        bankDetails.put("bank_name", "Hang Seng Bank Limited");
+
+        JSONObject beneficiary = new JSONObject();
+        beneficiary.put("additional_info", additionalInfo);
+        beneficiary.put("address", address);
+        beneficiary.put("bank_details", bankDetails);
+        beneficiary.put("date_of_birth", "1976-08-26");
+        beneficiary.put("entity_type", "PERSONAL");
+        beneficiary.put("first_name", "John");
+        beneficiary.put("last_name", "Walker");
+
+        return beneficiary;
+    }
+
+    @Deprecated
+    private JSONObject conductPayer() {
+        // TODO Modify the arguments, which should be passed from the client side.
+        JSONObject additionalInfo = new JSONObject();
+        additionalInfo.put("business_registration_number", "EU300503");
+
+        JSONObject address = new JSONObject();
+        address.put("city", "Melbourne");
+        address.put("country_code", "AU");
+        address.put("postcode", "3000");
+        address.put("state", "VIC");
+        address.put("street_address", "15 Williams Street");
+
+        JSONObject payer = new JSONObject();
+        payer.put("additional_info", additionalInfo);
+        payer.put("address", address);
+        payer.put("company_name", "Complete Concrete Pty Ltd");
+        payer.put("entity_type", "COMPANY");
+        return payer;
+    }
+
+    /**
+     * Query all the available balances in our account.
+     */
+    private JsonNode getBalanceInAirwallex() throws UnirestException {
+        HttpResponse<JsonNode> response = Unirest.get(AirwallexConstant.API_GET_BALANCES)
+                .header("Authorization", "Bearer " + AIRWALLEX_TOKEN)
+                .asJson();
+        return response.getBody();
+    }
+
+    /**
+     * Return a guaranteed rate for the currency pair you are looking to transact in.
+     */
+    @AirwallexRequest
+    private void getQuote() throws UnirestException {
+        HttpResponse<JsonNode> response = Unirest.post(AirwallexConstant.API_CREATE_QUOTE)
+                .header("Authorization", "Bearer " + AIRWALLEX_TOKEN)
+                .header("Content-Type", "application/json")
+                .body(this.conductRequestBodyForCreatingQuote().toString())
+                .asJson();
+    }
+
+    private JSONObject conductRequestBodyForCreatingQuote() {
+        JSONObject jsonBody = new JSONObject();
+        jsonBody.put("buy_amount", 10000);
+        jsonBody.put("buy_currency", "AUD");
+        jsonBody.put("conversion_date", TimezoneUtil.getCurrentDateByYYYYMMDD());
```

Comment:
Is it possible to use the **Date** type here, so we don't need to convert between String and Date types?